File: libgomp/testsuite/libgomp.oacc-fortran/pr84955.f90
! { dg-do compile }

subroutine s
   integer :: i, j
   !$acc parallel loop tile(2,3)
   do i = 1, 10
      do j = 1, 10
         do
         end do
      end do
   end do
  !$acc end parallel loop
end subroutine s

File: libgomp/testsuite/libgomp.oacc-c-c++-common/pr84955.c
/* { dg-do compile }  */

int
main (void)
{
  int i, j;

#pragma acc parallel loop tile(2,3)
  for (i = 1; i < 10; i++)
    for (j = 1; j < 10; j++)
      for (;;)
	;

  return i + j;
}
